package.json

```diff
@@ -32,6 +32,8 @@
     "rxjs": "7.5.6"
   },
   "devDependencies": {
+    "@fullstacksjs/eslint-config": "8.6.0",
+    "@fullstacksjs/toolbox": "2.19.0",
```

Comment:
`@fullstacksjs/toolbox` comes with lots of helpful tested utilities with 0 dependencies, we can take advantage of it.